src/main/java/com/diffblue/javademo/tictactoe/Player.java

```diff
@@ -0,0 +1,5 @@
+package com.diffblue.javademo.tictactoe;
+
+public enum Player {
+  O, X, empty;
```

Comment:
Surely `Player`'s can't be empty, only squares on the board can. Could this have a better name?